network/network.go

```diff
@@ -96,7 +96,14 @@ func (r Reachability) String() string {
 	return str[r]
 }
 
-// Stat stores metadata pertaining to a given Stream/Conn.
+// ConnectionStat stores metadata pertaining to a given Conn.
+type ConnectionStat struct {
```

Comment:
Let's call it `ConnStat` for consistency?

Reply:
I'd love to, but `ConnStat` is already the interface: https://github.com/libp2p/go-libp2p-core/blob/975f71b238330cbfafb4a96f674ff047686dc219/network/conn.go#L63-L67